tasks.rb

```diff
@@ -370,6 +372,10 @@ def create_hpxmls
     'base-pv-battery-lifetime-model.xml' => 'base-pv-battery.xml',
     'base-pv-battery-garage.xml' => 'base-enclosure-garage.xml',
     'base-schedules-simple.xml' => 'base.xml',
+    'base-schedules-detailed-setpoints.xml' => 'base.xml',
```

Comment:
Should we create one more sample file (base-schedules-detailed-all.xml) that includes all possible schedule columns (occupancy and setpoints)?

Reply:
In addition to `base-schedules-detailed-setpoints-10-mins.xml` which already does that?

Reply:
Are we looking at the same thing? That file only references the setpoints.csv. I want a sample file that references two schedule CSVs (setpoints and occupancy). Also, we should probably rename the smooth and stochastic CSV files to have an 'occupancy-' prefix (so, `occupancy-smooth.csv`, for example). I like that you named all the setpoint CSVs using a 'setpoints-' prefix.

Reply:
See https://github.com/NREL/OpenStudio-HPXML/blob/detailed-hvac-setpoints-2/workflow/sample_files/base-schedules-detailed-setpoints-10-mins.xml#L14-L15.